.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: BUG
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+<!-- Running `python_code` helps to provide vital details -->
+
+Python Client version: X.Y.Z
+GDS version: X.Y.Z
+Neo4j version: X.Y.Z
```

Comment:
We might be interested in Python version too?

Reply:
good point :+1: